code/game/objects/items/contraband_vr.dm

```diff
@@ -36,12 +36,12 @@
 						/obj/item/device/pda/mime,
 						/obj/item/device/pda/syndicate,
 						/obj/item/mecha_parts/chassis/phazon,
-						/obj/item/mecha_parts/part/phazon_head,
-						/obj/item/mecha_parts/part/phazon_left_arm,
-						/obj/item/mecha_parts/part/phazon_left_leg,
-						/obj/item/mecha_parts/part/phazon_right_arm,
-						/obj/item/mecha_parts/part/phazon_right_leg,
-						/obj/item/mecha_parts/part/phazon_torso,
```

Comment:
I don't want phazon parts removed. The way the phazon used to work is you just put the parts together and it was done, which was weird, but worked. If it has circuits now, add them. Don't remove the phazon.